modules/ztp-sno-accelerated-ztp.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-accelerated-ztp_{context}"]
+= Accelerated provisioning of {ztp}
+
+:FeatureName: Accelerated provisioning of {ztp}
+include::snippets/technology-preview.adoc[]
+
+You can reduce the time taken for installation by using accelerated provisioning of {ztp} for {sno}. 
+Accelerated ZTP speeds up installation by applying Day 2 manifests derived from policies at an earlier stage. 
+
+== Activating accelerated ZTP
+
+You can activate accelerated ZTP using the `spec.clusters.clusterLabels.accelerated-ztp` label, as in the following example:
+
+.Example Accelerated ZTP `SiteConfig` CR.
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v2
+kind: SiteConfig
+metadata:
+  name: "example-sno"
+  namespace: "example-sno"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "ssh-rsa AAAA..."
+  clusters:
+  # ...
+    clusterLabels:
+        common: true
+        group-du-sno: ""
+        sites : "example-sno"
+        accelerated-ztp: full
+
+----
+
+You can use `accelerated-ztp: full` to fully automate the accelerated process. 
+{ztp} updates the `AgentClusterInstall` resource with a reference to the accelerated {ztp} `ConfigMap`, and includes policies extracted from {cgu-operator}, and acclerated ZTP job manifests. 
+
+If you use `accelerated-ztp: partial`, {ztp} does not include the accelerated job manifests, but includes policy-derived objects created during the cluster installation, as follows:
+
+* `PerformanceProfile.performance.openshift.io`
+* `Tuned.tuned.openshift.io`
+* `Namespace`
+* `CatalogSource.operators.coreos.com`
+* `ContainerRuntimeConfig.machineconfiguration.openshift.io`
+
+This partial acceleration reduces the number of reboots done by the node when applying resources of the kind `Performance Profile`, `Tuned` and `ContainerRuntimeConfig`. 
```

Comment:
```suggestion
This partial acceleration may reduce the number of reboots done by the node when applying resources of the kind `Performance Profile`, `Tuned` and `ContainerRuntimeConfig`. 
```
That's because it depends on many other moving parts. For example, the procedure only works when doing "bootstrap-in-place", but if someone installs OCP using another installation option - it will still reboot all the same. OCP can be installed many different ways...

Reply:
ok. updated as discussed and added note to mention "Assisted Installer" requirement.